app/routes/planet_routes.py

```diff
@@ -0,0 +1,73 @@
+from flask import Blueprint, jsonify, make_response, abort
+'''
+Defined a Planet class with the attributes id, name,
+and description, and moons. Also, Created a list of Planet instances.
+'''
+
+
+class Planet():
+    def __init__(self, id, name, description, moons):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.moons = moons
+
+    def to_json(self):
+        return {
+            "id": self.id,
+            "name": self.name,
+            "description": self.description,
+            "moons": self.moons
+        }
+
+planets = [
+    Planet(1, "Mercury", ["Grey", "closest to the sun", "smallest planet"], False),
+    Planet(2, "Venus", ["Brown and grey", "hottest planet"], False),
+    Planet(3, "Earth", ["Blue, brown green and white", "water world","1 moon"], True),
+    Planet(4, "Mars", ["Red, brown and tan", "2 moons"], True),
+    Planet(5, "Jupiter", ["Brown, orange and tan, with white cloud stripes","largest planet", "79 moons"], True),
+    Planet(6, "Saturn", ["Golden, brown, and blue-grey"," large and distinct ring system" ,"82 moons"], True),
+    Planet(7, "Uranus", ["Blue-green", " holds the record for the coldest temperature ever measured in the solar system ","27 moons"], True),
+    Planet(8, "Neptune", ["Blue"," on average the coldest planet" ,"14 moons"], True)
+]
+
+planet_bp = Blueprint("planet_bp", __name__, url_prefix="/planets")
+
+#Get all planets
+@planet_bp.route("", methods=["GET"])
+def read_all_planets():
```

Comment:
Nice job naming the method to describe what this route does.

Since you put your routes in planet_routs.py in routes directory, you can even call this read_all() since we know that all the routes in planet_routes.py are related to the Planet class.